activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -11,9 +11,9 @@ module QueryMethods
     include ActiveModel::ForbiddenAttributesProtection
 
     # WhereChain objects act as placeholder for queries in which #where does not have any parameter.
-    # In this case, #where must be chained with #not to return a new relation.
+    # In this case, #where can be chained to return a new relation.
```

Comment:
Same problem with linking `#where` here too.  In this case, I think we can either prefix them with `QueryMethods`, or simply change `#where` to `+where+` with the assumption that the reader already has enough context to know what "`where`" refers to.